.travis.yml

```diff
@@ -34,6 +34,15 @@ matrix:
             - WEB_HOST="http://web"
             - APP_DEBUG=1
             - ENABLE_SYMFONY_CACHE=1
+        - name: "PHP 7.3 Symfony Cache integration tests with translation-aware cache enabled"
```

Comment:
it's worth to point out that Travis is not available for 4.0 and we're moving every CI to GitHub Actions, also for 3.3.

Reply:
Yep, I'm moving this job right now (https://github.com/ezsystems/ezplatform-http-cache/pull/165 was merged today)